frontend/packages/client/src/components/ProposalInformation.js

```diff
@@ -258,28 +310,11 @@ const ProposalInformation = ({
           />
           <InfoBlock
             title={'Author'}
-            content={
-              <div className="columns is-mobile m-0">
-                <div className="column is-narrow is-flex is-align-items-center p-0">
-                  <Blockies
-                    seed={creatorAddr}
-                    size={6}
-                    scale={4}
-                    className="blockies"
-                  />
-                </div>
-                <div className="column px-2 py-0 is-flex flex-1 is-align-items-center">
-                  {creatorAddr}
```

Comment:
can't use something like:
```
max-width: 150px;
overflow: hidden;
white-space: nowrap;
text-overflow: ellipsis;
display: inline-block;
```
bc size depends on parent component and setting a max with at this level doesn't work, component needs to know parent size to shrink or not

Reply:
another strategy do this with text-overflow would be to set `maxWidth` dynamically with javascript using refs etc but I think truncating an address by removing the middle chunks is much better than removing the end (people often use the first/last four characters of an address as a quick identifier)